src/test/java/org/fundacionjala/trello/pages/menus/Header.java

```diff
@@ -66,4 +74,19 @@ public MenuProfile getMenuProfile() {
         action.click(profileButton);
         return new MenuProfile(driver);
     }
+
+    public Header openNotifications() {
+        action.waitForVisibility(notificationBtn);
+        action.click(notificationBtn);
+        return this;
+    }
+
+    public BoardPage selectBoardNotification(final String type, final String boardName) {
+        String notificationPath = String.format(NOTIFICATION_PATH, type, boardName);
+        By notifyLocator = By.xpath(notificationPath);
+        action.waitForElementLocated(notifyLocator);
+        WebElement notification = driver.findElement(notifyLocator);
+        notification.click();
```

Comment:
use the action class to perform this click

Reply:
Done